### ISSUE TITLE
Input arguments to modcmaes.fmin

### ISSUE THREAD

**Issue by nikohansen** [IOHprofiler/ModularCMAES#32]
From the example in the readme
```python
modularcmaes.fmin(func=sum, dim=5, maxfun=100)
```
I wonder why the `fmin` function does not take an initial solution as input and what is the initial value used by `fmin` (or `modularcmaes.ModularCMAES` for that matter)?

It seems strange to me to assume a default initial solution value because it heavily depends on the problem and use-case. Similarly, I believe that the initial step-size `sigma` should be an input to the algorithm for which the user should be commandingly invited to make a decision, because it also depends on the problem and the desired use-case.

**Comment by jacobdenobel** [comment]
Thank you for your comment. We know that documentation is not yet up to standards, and we indeed only have included very minimal usage examples in the readme. We are planning to improve this in the future.

In principle, these and many more parameters can be passed as keyword arguments to both the class and the ```fmin``` based ways of calling the method. 

For changing the value for sigma, you could pass the option ```sigma0=...``` for example, and ```x0``` controls the initial setting of the centre of mass. 

Hope this helps.

**Comment by nikohansen** [comment]
> Thank you for your comment. We know that documentation is not yet up to standards, and we indeed only have included very minimal usage examples in the readme. We are planning to improve this in the future.
> 
> In principle, these and many more parameters can be passed as keyword arguments to both the class and the `fmin` based ways of calling the method.

Right, my point though would be to a lesser extend about which optional parameters can be passed if desired. I rather wanted to raise the question which parameters should be exposed and mandatory to pass in order to best reflect the semantics of the algorithm.
